packages/mobile/src/screens/track-screen/TrackScreenDetailsTile.tsx

```diff
@@ -461,7 +461,12 @@ export const TrackScreenDetailsTile = ({
   ])
 
   const renderBottomContent = () => {
-    return hasDownloadableAssets ? <DownloadSection trackId={trackId} /> : null
+    return hasDownloadableAssets ? (
+      <>
+        <Divider />
+        <DownloadSection trackId={trackId} />
+      </>
+    ) : null
   }
```

Comment:
Whats this? Feels unrelated?

Reply:
This is a fun one. In a previous PR, I removed the `overflow: hidden` from the `TrackDetailsTile` because it was hiding the shadow, which was very noticeable next to the shadow of the comment section. However, there was a divider that was always being rendered, regardless of whether the download section was shown. This divider was also being hidden by the `overflow: hidden`. So, this change conditionally renders the divider only when needed